book/04-git-server/sections/generating-ssh-key.asc

```diff
@@ -1,4 +1,4 @@
-[[r_generate_ssh_key]]
+[[_generate_ssh_key]]
```

Comment:
Please don't change this anchor. It is referred on other parts of the text and would broke internal links.

Reply:
@jnavila I understand, but I've changed this to try correcting the failing build. I'll revert all of the commits in which I've changed anchors, which should bring the PR to the state it was in originally.